src/DataSource/DB.hs

```diff
@@ -147,7 +149,13 @@ connDBLayer = DBLayer
 
 -- | The connection pooled Zendesk layer. Used for database querying.
 -- We need to sync occasionaly.
-connPoolDataLayer :: forall m. (MonadBaseControl IO m, MonadIO m, MonadReader Config m) => DBConnPool -> DataLayer m
+connPoolDataLayer :: forall m. (MonadBaseControl IO m
```

Comment:
This formatting is wrong. Not sure if this is `stylish-haskell` or `hindent` or something else, but it should look like this:
```
connPoolDataLayer 
    :: forall m. (MonadBaseControl IO m, MonadIO m, MonadReader Config m, MonadCatch m)
    => DBConnPool
    -> DataLayer m
```
Or if you want to split it into newlines (the max width should be 100, which is not really upheld in this project, we can fix that easily):
```
connPoolDataLayer 
    :: forall m. 
    ( MonadBaseControl IO m
    , MonadIO m
    , MonadReader Config m
    , MonadCatch m
    )
    => DBConnPool
    -> DataLayer m
```

Reply:
I did not this. Thank you for telling me!